### PR TITLE
chore: bump versions

### PR DESCRIPTION
**Pull request by qued**
Update versions of dependencies, including unpinning the `unstructured-inference` dependency that's causing conflicts in repos like `pipeline-oer` that want the newer version.

#### Testing:
CI should pass.

### CODE REVIEW

**Review by MthwRobinson — APPROVED** [review]
Bump day! 🐪 
LGTM!

File: test_unstructured_ingest/test-ingest-s3.sh [hunk 1]
```diff
@@ -11,13 +11,13 @@ fi
...
-if ! diff -ru s3-small-batch-output test_unstructured_ingest/expected-structured-output/s3-small-batch ; then
+if ! diff -ru test_unstructured_ingest/expected-structured-output/s3-small-batch s3-small-batch-output ; then
     echo
     echo "There are differences from the previously checked-in structured outputs."
-    echo
+    echo 
     echo "If these differences are acceptable, copy the outputs from"
     echo "s3-small-batch-output/ to test_unstructured_ingest/expected-structured-output/s3-small-batch/ after running"
-    echo
+    echo
```
Comment (cragwolfe) [inline test_unstructured_ingest/test-ingest-s3.sh:20]: added whitespace?

**Comment by cragwolfe** [comment]
Nicely done! @qued , what is the secret to match the outputs in a local env? (and hopefully also achievable on mac)

**Comment by qued** [comment]
@cragwolfe In an Ubuntu 22.04 instance that isn't running the setup script, I ran the same setup commands in the same order that CI does:
- `apt update`
- Install git
- Install `pyenv` and Python 3.8.15
- Clone repo and create and activate virtualenv
- `make install-ci`
- `make install-detectron2`
- `sudo apt-get install -y libmagic-dev poppler-utils tesseract-ocr libreoffice`
- `make test` (Just to make sure)
- `make install-ingest-s3`
- `make install-ingest-azure`
- `make install-ingest-github`
- `make install-ingest-gitlab`
- `make install-ingest-wikipedia`
- `./test_unstructured_ingest/test-ingest.sh`

I'm going to go ahead and merge this but I'll add an issue to investigate what causes the deviations from the output.

**Pull request merged**